3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,85 @@
+//Flight Booking Fullname
+function fullName(firstName, surName) {
+  return firstName + " " + surName;
+}
+const fullName1 = fullName("Anjali", "Kumari");
+const fullName2 = fullName("Babul", "Raja");
+console.log(fullName1);
+console.log(fullName2);
+
+//Formal Name
+function getFullname(firstName, surName, useFormalName, isWoman) {
+  if (useFormalName) {
+    return "Lord" + " " + firstName + " " + surName;
+  } else if (isWoman) {
```

Comment:
The condition  `if(isWoman)` should be nested inside the first condition `if(useFormalName)`.
according to your code, the following cases happen:

```suggestion
  } const formalName1 = getFullname("Anjali", "Kumari", false, true);//1 Miss Anjali Kumari
const formalName2 = getFullname("Anjali", "Kumari", true, true);//2 Lord Anjali Kumari
const formalName3 = getFullname("Anjali", "Kumari", false, false);//3 Anjali Kumari
const formalName4 = getFullname("Anjali", "Kumari", true, false);//4 Lord Anjali Kumari {
```
The first and second one are wrong.